jac-uk/apply
# Feature/1040 application form improvements

## What's included?
Closes #1040

Design: https://www.figma.com/proto/5XatHx18Pabno2dGrgeYtS/%231040---Application-form-improvements?page-id=0%3A1&type=design&node-id=4-3684&viewport=199%2C308%2C0.13&t=NiGFwt0GsOeMcBAq-1&scaling=min-zoom&starting-point-node-id=4%3A3684&mode=design

Note: The changes will apply to `_applicationVersion 3`.

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
Example vacancy: 
- https://jac-apply-develop--pr1073-feature-1040-applica-ftrcoggj.web.app/vacancy/ONqUjAzhDS7rjh6dFJLB/
- https://jac-apply-develop--pr1073-feature-1040-applica-ftrcoggj.web.app/vacancy/Y1gSQnCMhbqpA0nHmDaM/

Number of days required experience: 1000

1. Apply for a vacancy.
2. Go to "Previous judicial experience".
3. Check if the form works the same as the design.

Note: The number of days required experience can be set up on admin, e.g.

![image](https://github.com/jac-uk/apply/assets/79906532/40468e46-06d2-47eb-acbf-6578530a1a92)


## Risk - how likely is this to impact other areas?
🟠 Medium risk - this does change code that is shared with other areas

## Additional context
Include screen grabs, notes etc.

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
